tests/base.py

```diff
@@ -0,0 +1,11 @@
+import maya.cmds as mc
+from unittest import TestCase
+
+
+class BaseTestCase(TestCase):
+    @classmethod
+    def setUpClass(self):
```

Comment:
I believe this should be `setUp` instead of `setUpClass`. The way I wrote a couple of the tests it should be safer to reset before every method, rather than every class.

Feel free to leave this as is though. There would be the need to add a `super(XYZ, self).setUp()` to all the classes with their setUp. I could take part of these annoying tasks :)

Reply:
I used setUpClass so it would just make sure that the first test run in the TestCase suite would be working from a new scene, just in case whatever TestCase that was running before it didn't clean up after itself.  I figured it would be overkill to do both setup and teardowns for new scene creation.  I can update it to setUp instead if you'd like though!